### DISCUSSION TITLE
Running task on same instance (same host name workers)

### DISCUSSION THREAD

**Discussion by kennyth01** [apache/airflow#19975] [Q&A]
Hi guys,
I have a dag file that has two tasks named t1 and t2

`t1 >> t2`

Now i have an issue, assuming I have several workers, 
sometimes t1 is running on task instance1 and t2 is running on task instance2

How can i achieve both task should run on same task instance host(worker server)?

Thanks in advance.

**Comment by potiuk** [comment]
You could assign the worker to a queue (if you use celery) and have both tasks use that queue.

**Comment by potiuk** [comment]
(It will limit your scalability for those tasks as they will only be able to run on the single worker that is in the queue)

**Comment by kennyth01** [comment]
Hello guys, im on a MWAA setup though :( Another question, iis there a work around wherein i want to run my dag that as multiple task in the same worker?

i have a bash operator task and ssh operator task in my DAG, and i want them to run on the same instance, anyway i can achieve this?
